### PR TITLE
huiswerk-push-one

### PR DESCRIPTION
**Pull request by NielsLouwes**
Leave a comment to inform people in your team about changes etc.

THings to consider.

### CODE REVIEW

**Review by NielsLouwes — COMMENTED** [review]
All good.

File: index.html [hunk 1]
```diff
@@ -6,6 +6,6 @@
     <title>Nep huiswerkopdracht</title>
 </head>
 <body>
-
+<p>Dit is normale text voor het huiswerk.</p>
```
Comment (nteulings) [inline index.html:9]: Lijkt me prima!

**Review by nteulings — CHANGES_REQUESTED** [review]
Goed gedaan Niels!

**Comment by TheoSnoek** [comment]
lekker bezig

**Pull request merged**